ldoc/markup.lua

```diff
@@ -254,12 +255,13 @@ local formatters =
                   end
                else
                   io.stderr:write('LDoc discount failed with error ',errmsg)
-                  io.exit(1)
+                  os.exit(1)
```

Comment:
nice one

Reply:
There were and are some real head-scratchers in this code base. This one evidently worked as expected because trying to call a non-existent function would have had the same effect as deliberately exiting with an error.